Lesson27/gozde/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Posts</title>
+		<link rel="stylesheet" type="text/css" href="styles.css" />
+	</head>
+	<body>
+		<div class="container">
+			<h1>Posts</h1>
+			<button id="fetch-button" class="button button--success">
+				Fetch posts
+			</button>
+		
+			<button id="create-button" class="button button--success">
+				Create post
+			</button>
+			<!-- TODO: update and delete buttons should be moved to every post that is shown on the page -->
+			<button id="update-button" class="button button--success">
+				Update post
+			</button>
+			<button id="delete-button" class="button button--danger">
```

Comment:
Remove this button. Delete buttons should be added on each post (otherwise there's no selected post to delete).